resources/stylesheets/page/_register.scss

```diff
@@ -0,0 +1,105 @@
+*{
+    padding: 0;
+    margin: 0;
+}
```

Comment:
this style is apply to all the elements so it goes to the base folder

Reply:
Done